apps/koala-frontend/src/app/features/sessions/pages/session-analysis/session-analysis.page.ts

```diff
@@ -1,10 +1,260 @@
-import { Component } from '@angular/core';
+import { ApplicationRef, Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { AuthService } from '../../../auth/services/auth.service';
+import { MediaControlService, MediaEvent, MediaActions } from '../../services/media-control.service';
+import { MessageService } from 'primeng/api';
+import { SessionsService } from '../../services/sessions.service';
+import { AnnotationService } from '../../services/annotation.service';
+import { MarkerService } from '../../services/marker.service';
+import { TranslateService } from '@ngx-translate/core';
+import { DataPoint, Display } from '../../components/annotation/annotation.component';
+import { Session } from '../../types/session.entity';
+import { environment } from '../../../../../environments/environment';
+import { Marker } from '../../types/marker.entity';
+import { filter } from 'rxjs';
+
+export interface AnnotationData {
+  AnnotationData: Map<number, Array<DataPoint>>;
+}
 
 @Component({
-  selector: 'koala-session-analysis',
+  selector: 'koala-app-session-analysis.page',
```

Comment:
what's the reason for the selector change? The pattern in our other components is koala-<name of component without the 'page' suffix> ... is there any check running on this?